internal/runners/packages/searchresult.go

```diff
@@ -0,0 +1,71 @@
+package packages
+
+import (
+	"strings"
+
+	"github.com/ActiveState/cli/internal/output"
+	"github.com/ActiveState/cli/internal/rtutils/ptr"
+	"github.com/ActiveState/cli/pkg/platform/model"
+)
+
+type structuredSearchResults struct {
+	Results      []*searchResult `locale:"," opts:"verticalTable" json:"Results,omitempty"`
+	packageNames []string
+}
+
+type searchResult struct {
+	Name            string         `opts:"omitEmpty" locale:"search_name, [HEADING]Name[/RESET]" json:"Name,omitempty"`
+	Description     string         `opts:"omitEmpty" locale:"search_description, [HEADING]Description[/RESET]" json:"Description,omitempty"`
+	Website         string         `opts:"omitEmpty" locale:"search_website, [HEADING]Website[/RESET]" json:"Website,omitempty"`
+	License         string         `opts:"omitEmpty" locale:"search_License, [HEADING]License[/RESET]" json:"License,omitempty"`
+	Versions        []string       `opts:"omitEmpty" locale:"search_versions, [HEADING]Versions[/RESET]" json:"Versions,omitempty"`
+	Vulnerabilities map[string]int `opts:"omitEmpty" locale:"search_vulnerabilities, [HEADING]Vulnerabilities[/RESET]" json:"Vulnerabilities,omitempty"`
+	version         string
+}
+
+func createSearchResults(packages []*model.IngredientAndVersion, vulns []*model.VulnerabilityIngredient) (*structuredSearchResults, error) {
+	var results []*searchResult
+	var packageNames []string
+	for _, pkg := range packages {
+		result := &searchResult{}
+		result.Name = ptr.From(pkg.Ingredient.Name, "")
+		result.Description = ptr.From(pkg.Ingredient.Description, "")
+		result.Website = pkg.Ingredient.Website.String()
+		result.License = ptr.From(pkg.LatestVersion.LicenseExpression, "")
```

Comment:
Author information is missing.

Reply:
I should have noted this somewhere but we discussed during stand that retrieving the author information requires us to make a separate API call for each package in the search results. We deemed that this would slow down the command too much so opted to leave the authors out.

Reply:
Thanks, forgot about that. I filed DX-2581 and a blocker with XT to hopefully get this resolved.